es-design-system/pages/organisms/es-reviews-list.vue

```diff
@@ -41,8 +41,10 @@
                         :developer-logo="review.developer_response ? review.developer_response.developer_logo : null"
                         :response-date="review.developer_response ?
                             new Date(review.developer_response.modified) : null"
+                        :report-flag-visible="index==0 || index==2"
```

Comment:
Again, instead of hardcoding this, we want to set the value of `reportFlagVisible` in the example review objects in the `data()` section of this file.